datadriven/examples/PDECombigridTest.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2008-today The SG++ project
```

Comment:
This should be in the examplesPipeline folder.

Is it not possible to execute this via the already existing UniversalMiner example?

Reply:
Yes I'll delete it.